src/App.js

```diff
@@ -21,95 +21,41 @@ const messages = {
 addLocaleData([...en, ...es]);
 
 class App extends Component {
-  constructor() {
-    super();
+  constructor(props) {
+    super(props);
+
+    // TODO: Consider continue determining here default instance or moving all upside,
```

Comment:
upside you mean index,js?

Reply:
Claro, por ahora la inyección de dependencias desde index es opcional, y se está haciendo en los tests.

Si no se inyecta, se define acá basado en las dependencias por defecto.

Que opinás de directamente mover la definición de las dependencias a index y que sea siempre requerida?

Reply:
no veo cual seria la contra de moverlas

Reply:
Pros de dejarla acá:

* Hace más fácil el uso por defecto
* Deja más claras cuales son las dependencias por defecto

Pros de moverlo afuera:

* Hace más predecible el uso del componente porque las dependencias siempre son requeridas
* Imagino que hace más difícil el intellisense al editar el archivo ya que los imports están afuera

Reply:
Por ahora lo dejo aquí, dejemos abierta la opción de moverlo afuera para después.